yarn.lock

```diff
@@ -6428,9 +6428,9 @@ globby@^11.0.2, globby@^11.0.4:
     merge2 "^1.3.0"
     slash "^3.0.0"
 
-"gluegun@git+https://github.com/edgeandnode/gluegun.git#v4.3.1-pin-colors-dep":
```

Comment:
Is this to fix the error in yarn lock file?

Reply:
Yes, when I pulled and did yarn install, I noticed the yarn lock file updated with this, so I committed it to see if it'll fix the current CI issue.